Sources/Commands/show-dependencies.swift

```diff
@@ -107,7 +107,13 @@ private final class JSONDumper: DependenciesDumper {
                 ])
         }
 
-        print(convert(rootpkg).toString(prettyPrint: true))
+        print(convert(rootpkg).toString(prettyPrint: true, contentsKeyOrder: contentsKeyOrder(a:b:)))
+    }
+
+    /// To bring `dependencies` key to the last order.
+    /// See: https://bugs.swift.org/browse/SR-5624
+    func contentsKeyOrder(a: String, b: String) -> Bool {
+        return b == "dependencies" ? a > b : a < b
```

Comment:
This appears to be incorrect as a sort predicate and only happens to work because of the particular keys hardcoded above. Consider what would happen if you had keys named "data", "dependencies", and "name".

A sort predicate that unconditionally moves `dependencies` last would be `{ $0 != "dependencies" || $1 == "dependencies" || $0 < $1 }`.